projects/core/styles/markup/tui-mobile-only.less

```diff
@@ -3,3 +3,13 @@
         display: none !important;
     }
 }
+
+@media @mobile-m {
+    html,
+    body {
+        // only for iOS devices
+        @supports (-webkit-touch-callout: none) {
+            overflow: auto;
```

Comment:
can we somehow encapsulate this logic inside `auto-focus`-directive ?
I am afraid that we can forget later that this line is closely connected with this directive(

(or add comment, at least?)

Reply:
fixed